packages/terra-graphs/src/components/Dev.module.scss

```diff
@@ -0,0 +1,140 @@
+body {
```

Comment:
We likely don't want to be adding body styles here. If they are necessary, we should add them to our base stylesheet.

Reply:
We have a Base component that is required to be used in conjunction with Terra components. It has a stylesheet that defines the global styles for the overall document: https://github.com/cerner/terra-core/blob/main/packages/terra-base/src/Base.scss

If we need these styles to be present in the body, we should add them to that Base stylesheet rather than in this one, otherwise they're likely to get lost. Though if these styles are only used for your examples (as far as I can tell), are they really necessary at all?

Reply:
can I add these to the base styleSheet, or is it something that needs to be done by someone?

Reply:
If they are absolutely necessary, log an issue to terra-core to have them added to terra-base: https://github.com/cerner/terra-core/issues

Reply:
Removed body [styling](https://github.com/cerner/terra-graphs/pull/106/commits/b812e6fee039c04a3f269ce40365a30bb2218209#diff-bfe4ff0ee38064d336a43752b43a47ebff4450d6ee264c91bc875a9b9aec6d5eL1) since it belongs to the old dev site.